src/main/java/com/question/solutions/concurrency/concurrency/ExecutorService.java

```diff
@@ -1,14 +1,13 @@
-package com.jthomas.concurrency;
+package com.question.solutions.concurrency.concurrency;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.Callable;
 import java.util.concurrent.ExecutionException;
-import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
-public class JthomasExecutorService {
+public class ExecutorService {
```

Comment:
Can you also give a custom name for this?

Reply:
Can you please suggest names when you ask for a different name?